<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParabolicInterface</class>
 <widget class="QMainWindow" name="ParabolicInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParabolicInterface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="4">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>Explicit</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2" rowspan="5">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="5">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="6" column="1" colspan="4">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Solve</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="text">
       <string>Analytic</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Choose solution method</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QRadioButton" name="radioButton_3">
      <property name="text">
       <string>Crank-Nikolsn</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>Implicit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="text">
       <string>200</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="text">
       <string>3.14159</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="text">
       <string>15</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="text">
       <string>0.01</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>N = </string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>K = </string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>l = </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>T = </string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>a = </string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>sigma = </string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>teta = </string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QLineEdit" name="lineEdit_6"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
